app/scoring/page.tsx

```diff
@@ -1,9 +1,61 @@
-export default function Scoring() {
-    var currentSelectedContestant = "Jacob"
+import { getTeamList, ITeam } from "../utils/wikiQuery"
+import { wikiUrl, getWikipediaContestantData } from "../utils/wikiFetch"
 
+export default async function Scoring() {
+
+    const wikiContestants = await getWikipediaContestantData()
+    const pageData = getTeamList(wikiContestants)
+
+    const currentSelectedContestant = "Jacob"
+
+    const numberOfRounds = pageData.props.runners.reduce(
+        (acc: number, x: ITeam) => {
+            return x.eliminationOrder > acc ? x.eliminationOrder : acc
+        }, 0)
+
+    const roundScores = []
+    for(let i = 1; i <= numberOfRounds; i++) {
+        const roundScore = pageData.props.runners.reduce(
+            (acc: number, x: ITeam) => {
+                return x.eliminationOrder === 0 || x.eliminationOrder > i ? acc + 10 : acc
+            }, 0)
+        roundScores.push(roundScore)
+    }
+
+    const weeklyScore = pageData.props.runners.reduce(
+        (acc: number, x: ITeam) => {
+            return x.isParticipating ? acc + 10 : acc
+        }, 0)
+
+    const reverseTeamsList = [...pageData.props.runners].reverse()
+    let grandTotal = 0
+    let currentWeek = 0
     return (
         <div>
             <h1 className="text-2xl text-center">Current Scoring for {currentSelectedContestant}</h1>
+            <br/>
+            <div className="text-center">
+                {roundScores.map(s => {
```

Comment:
Personal preference, but I kinda like writing out `score` here, especially since line 47 uses `<s>` to strike-through text. Using `s` is perfectly valid here, though, just wanted to float the idea.

Reply:
Totally love that, will make the change. I generally start with just using short variables for lambdas, but this is quite a big lambda so 👍